README.md

```diff
@@ -8,19 +8,26 @@ Inspired by Doomspork's [Elixir companies][elixir-companies] list.
 
 [elixir-companies]: https://github.com/doomspork/elixir-companies
 
+#### Advertising
+
+* [Honeycomb](https://honeycomb.tv)
+  ([GitHub](https://github.com/honeycomb-tv)) - Faster delivery and better
```

Comment:
Maybe mention the video delivery stuff, just delivery sounds too ambiguous after all it could be that we deliver pizzas lol

Reply:
I took it from the website. Perhaps we should prefix "supercharged video advertising delivery", or whatever it is.

It is in the advertising section, so perhaps it's clear enough?

Reply:
Possibly yes to the prefix... wow I think our website might be a little too ambiguous because the 2nd part of that sentence difinitely feels very jargon laced

Reply:
Don't tell marketing! :P